<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QMainWindow" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="repoList"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="descriptionLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="computeButton">
        <property name="text">
         <string>Compute Total</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="calculateLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="listPrecipitation"/>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="precipitationLabel">
          <property name="text">
           <string>Precipitation</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="precipitationLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="filterButton">
        <property name="text">
         <string>Filter</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>706</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>filterButton</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>filterCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>computeButton</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>hoursCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>filterCommand()</slot>
  <slot>hoursCommand()</slot>
 </slots>
</ui>
